### ISSUE TITLE
renamed xboxapp xkm not detecting

### ISSUE THREAD

**Issue by shrek88** [fqlx/XboxKeyboardMouse#158]
hey got a problem since couple of days when microsoft decided to uppgrade the xboxapps name,  now it dosnt know it is running. and just stays blue. ive looked abit on my own but i cannot find where to change or so please fix this if u can please

**Comment by RenanBan** [comment]
yeah, this new xbox app update, the mouse and keyboard app doesnt lock the mouse into the app, need fix asap

**Comment by secretspy911** [comment]
It is really easy to fix on our end until someone from the projects fixes it. Just go in the XboxStream class and change line 32 for the right name. I did it last night and it works.

**Comment by RenanBan** [comment]
Wish i knew how to do that

**Comment by RenanBan** [comment]
Could you give a quick tour on how to do that, you change on github? or on the computer itself?

**Comment by secretspy911** [comment]
You cannot do it on github unless you are a contributor. Do it locally on your computer. Git clone the project (or download the zip) on your computer, open the code (you will need to download Microsoft Visual Studio 2019 if you don't have it). Change the code and run it from Visual Studio.

**Comment by RenanBan** [comment]
hmm nice, thanks a lot man will do

**Comment by secretspy911** [comment]
But what sucks now is the name of the app is not longer language neutral. The name before was "Xbox" which is the same in all the languages. Now, for me in French I had to write "Compagnon de la console Xbox". So now the project will only work with English computers, unless the project contributors can find something to handle it.

**Comment by RenanBan** [comment]
Thats bad, the guys will figure it out im sure, and i managed to get it working, thanks for the help

**Comment by shrek88** [comment]
@secretspy911  how should it say im in a english version it says Xbox Console Companion but it still wont trigger :( should spaces in the name be just spaces or no spaces or ? love if u sent me the bit of text edited here in commment that i need to change

**Comment by secretspy911** [comment]
Whatever the language, the text in the code must be the exact same the the text you see at the top of the xbox app when it's open.

![image](https://user-images.githubusercontent.com/50241396/59040034-ea80b280-8843-11e9-82b7-7a82b3c78c1b.png)

**Comment by shrek88** [comment]
nvm Just got it to work i hope xD mouse locked in window app turned green XD thanks alot

**Comment by shrek88** [comment]
solved issue and i got a build that works if ur OS is english and can mabey? be sent? dunt know-.- thankks @secretspy911 (had solved the issue long ago haha but missspelled the name :D thanks for correcting me) ^^

**Comment by DavidRieman** [comment]
Yeah we should get some code in that detects the app better. Its the same exact string match that blocked an Xbox app beta version user a while ago.  Then we can push a new click-once update to auto deploy the fix to everyone.

Im pretty busy but will try to squeeze out some time to start looking into this late tonight.

**Comment by DavidRieman** [comment]
reopening until we deploy a fix to everyone

**Comment by shrek88** [comment]
I can could send u the change I made for English OS if you want  I have it in a .rar  .zip file just don't know alot of how GitHub, works just started trying to "code" and yea

Den 6 juni 2019 19:37 skrev David Rieman <notifications@github.com>:

reopening until we deploy a fix to everyone

**Comment by DavidRieman** [comment]
@shrek88 don't worry, we got this.

**Comment by kostkol87** [comment]
![image](https://user-images.githubusercontent.com/8214416/59160106-8b889b00-8ada-11e9-8681-88164692a9b4.png)
seems like title already there, but it still does not trigger mouse =((

**Comment by DavidRieman** [comment]
@kostkol87 you can try opening the .sln file in Visual Studio and building it that way.  (`setdd` isn't recognized on my Win10 machine either, and there are other oddities in `build.bat` like the configuration being out of date.  I know I haven't ever touched that `build.bat` and we should probably delete it or fix it.)

Or wait for the fixed binaries, which I'm working on soon.

**Comment by DavidRieman** [comment]
Great news:  I've deployed the new version.  If you've used the [installation instructions](https://github.com/fqlx/XboxKeyboardMouse#installation) to install with that click-once installer, then the next launch should auto-update to the fixed version.  Please try it out and report back here so we can confirm and close the issue.

**Comment by CrsDany** [comment]
> Great news: I've deployed the new version. If you've used the [installation instructions](https://github.com/fqlx/XboxKeyboardMouse#installation) to install with that click-once installer, then the next launch should auto-update to the fixed version. Please try it out and report back here so we can confirm and close the issue.

Still not working, how can I resolve?
